### PR TITLE
Added some extra steps to Debian

### PR DESCRIPTION
**Pull request by tchellomello**
Adding some steps to debian/ubuntu

### CODE REVIEW

File: ansible/tasks/_prep-create-devops-user-sudo.yml [hunk 1]
```diff
@@ -2,14 +2,33 @@
...
-# ansible-playbook -u root --tags="prereq_as_root" main.yaml
+# ansible-playbook -u root --tags="prereq_as_root" main.yml
+#
+  - name: Install sudo
+    apt:
+      name: sudo
+      state: latest
+    when: ansible_distribution_file_variety == "Debian"
+    tags:
+      - prereq_as_root
```
Comment (phess) [inline ansible/tasks/_prep-create-devops-user-sudo.yml]: Installing sudo should be a mandatory task on non-debian systems as well.

**Review by phess — CHANGES_REQUESTED** [review]
Please add sudo installation to non-debian systems as well.

**Comment by tchellomello** [comment]
@phess  code updated

**Review by phess — APPROVED** [review]
Just beautiful.

**Pull request merged**